### PR TITLE
feat(docs): Install support and new blog

### PR DESCRIPTION
**Pull request by kris-nova**
Refactor from the last Falco call regarding install methods

Signed-off-by: Kris Nova <kris@nivenly.com>

**What type of PR is this?**

> Uncomment one (or more) `/kind <>` lines:

> /kind bug

/kind cleanup

> /kind design

> /kind user-interface

> /kind content

> /kind translation

**Any specific area of the project related to this PR?**

> Uncomment one (or more) `/area <>` lines:

/area blog

/area documentation

**What this PR does / why we need it**:

This updates the website with some much-needed changes around official support. 

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

**Comment by kris-nova** [comment]
I can demo this on the next call if needed

**Comment by leogr** [comment]
> I can demo this on the next call if needed

Yes, please!

**Comment by leogr** [comment]
Hey @kris-nova 

Since https://github.com/falcosecurity/falco/pull/1172 has been merged, we should add instructions about how to use the [driverloader docker image](https://github.com/falcosecurity/falco/issues/1171).
Using that is really easy:
```shell
docker run --rm -i -t -v /dev:/host/dev -v /proc:/host/proc:ro -v /boot:/host/boot:ro -v /lib/modules:/host/lib/modules:ro -v /usr:/host/usr:ro falcosecurity/driverloader
```
I could add that to the documentation, but I'm not sure which is the right place. 

Also:
- I think https://github.com/falcosecurity/falco/pull/1157 is blocked by this PR (we can't remove resources from the main repo if the documentation references to them)
- Before adding any further content, this PR should be rebased on top of the current master since there're important updates

Finally, the next Falco release is going to happen within 2 weeks, thus I believe this PR is becoming a high priority now.

Let me know how I can help out.

**Comment by leogr** [comment]
See also:
https://github.com/falcosecurity/falco/issues/1120
https://github.com/falcosecurity/falco/issues/1085

**Comment by leogr** [comment]
I'm working in this PR as discussed
/hold

**Comment by leogr** [comment]
Please review
/cc @fntlnz 
/cc @kris-nova 
/cc @leodido

**Comment by maxgio92** [comment]
I started reading from the deploy preview and I'm wondering if there's a way to add anchors to the titles. Just a little idea :)

**Comment by leogr** [comment]
Some are already present, manually added (see #docker).

File: content/en/docs/running.md [hunk 1]
```diff
@@ -1,156 +1,155 @@
...
-kind create cluster --config=./kind-config.yaml
-```
-
-4. [Install](../installation) Falco in your Kubernetes cluster with kind.
 
-## Running Falco manually
+## Run Falco manually
 
-If you'd like to run Falco by hand, here's the full usage description for falco:
+If you'd like to run Falco by hand, here's the full usage description for Falco:
```
Comment (kris-nova) [inline content/en/docs/running.md:24]: I generally do not suggest adding things like this to the documentation. This means every time we change something in the code we have to remember to change it here. Also this isn't relevant to all versions of Falco.

I would rather see something like:

The Falco help menu that is specific to your version of Falco can be found by

```
falco --help
```

File: content/en/docs/running.md [hunk 2]
```diff
@@ -1,156 +1,155 @@
...
-Falco is meant to be run as a service. But for experimentation and designing/testing rulesets, you will likely want to run it manually from the command-line.
 
-## Running Falco as a service
+## Run Falco as a service
 
-Once you've [installed](../installation) Falco as a package, you can start the service:
+If you installed Falco by using [the deb or the rpm](../installation) package, you can start the service:
 
 ```bash
 service falco start
```
Comment (kris-nova) [inline content/en/docs/running.md:13]: Can we also add a note that says that `systemctl` will also work even though the Falco service is configured with init.d

```
systemctl start falco
```

File: content/en/docs/download.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+{{< info >}}
+
+Falco depends on having a driver installed on the host system to get information about the running system calls.
+
+The preferred installation method is to install the driver using the native artifacts defined above or 
+temporarily run the `falcosecurity/falco-driver-loader` image as privileged, then using the `falcosecurity/falco-no-driver`.
+
+{{< /info >}}
+
+|tag | pull command | description |
```
Comment (kris-nova) [inline content/en/docs/download.md:40]: Can we call out which containers require privileges to escalate to the host? I hate to nit-pick this but it really is a big deal for a lot of production users. Maybe we just add a column at the end `requires-privileged` or something?

**Comment by kris-nova** [comment]
/approve 

Can I approve a pr I started but leo finished? i have no idea

File: content/en/docs/running.md (hunk 2 above)
Comment (leogr) [reply content/en/docs/running.md:13]: We have to test if that works properly both on deb and rpm. Once the test has been done, I will add it.

**Comment by fntlnz** [comment]
In the `CentOS/RHEL/EC2 Linux/Fedora` section under install I would also like to add Amazon Linux 2 and 1 since we actively test them before each release

**Comment by fntlnz** [comment]
Wrapping this up to later add to this document:

For the systemd units, it actually works because does is using the systemd-sysv-generator to wrap init.d scripts into systemd units.

Read more here:
- https://www.freedesktop.org/software/systemd/man/systemd-sysv-generator.html
- https://github.com/systemd/systemd/blob/master/src/sysv-generator/sysv-generator.c

**Comment by fntlnz** [comment]
Approved but adding an hold to merge this at the exact moment we cut the release.

File: content/en/docs/download.md (hunk 3 above)
Comment (leogr) [reply content/en/docs/download.md:40]: After having talked with @leodido and @fntlnz last week, I believe that just a simple "privileged YES/NO" is not enough (for example, see [here](https://github.com/falcosecurity/falco-website/pull/184/files#diff-d7e08ba2003658dbc0eac816ae6939c4R114-R146)). 
We have to well explain to users how to use all images, IMHO.
Maybe we need a futher PR to improve that.

Comment (leogr) [reply content/en/docs/download.md:40]: See also https://github.com/falcosecurity/falco-website/issues/196

File: content/en/docs/running.md (hunk 2 above)
Comment (leogr) [reply content/en/docs/running.md:13]: :point_right: https://github.com/falcosecurity/falco-website/pull/184#issuecomment-630033576

File: content/en/docs/installation.md [hunk 4]
```diff
@@ -1,353 +1,138 @@
...
-```
+4. Uninstall Falco:
 
-### Package install {#package}
+    ```shell
+    apt-get remove falco
+    ```
 
-#### CentOS/RHEL/Amazon Linux {#centos-rhel}
+### CentOS/RHEL/EC2 Linux/Fedora {#centos-rhel}
```
Comment (leogr) [inline content/en/docs/installation.md:62]:
```suggestion
### CentOS/RHEL/Fedora/Amazon Linux 2 {#centos-rhel}
```

**Pull request merged**